default.build

```diff
@@ -319,23 +319,12 @@
 			<in>
 				<items>
 					<include name="${nuget.nupackages.dir}/*.nupkg"/>
-					<exclude name="${nuget.nupackages.dir}/*.symbols.nupkg"/>
 				</items>
 			</in>
 			<do>
 				<echo message="nuget push -source https://nuget.org/ ${path::get-file-name(filename)} ${environment::newline()}" file="${nuget.nupackages.pushbatfile}" append="true"/>
```

Comment:
This url is not correct for pushing snupkg. They are ignored on it, even when explicitly pushing them.
I am going to PR the required url for pushing snupkg along with the regular package.
Seen after publishing NHibernate.Caches 5.6, done nhibernate/NHibernate-Caches#74 for fixing it there.